conf/defaults.ini

```diff
@@ -1533,6 +1533,8 @@ public_key_retrieval_on_startup = false
 disable_plugins =
 # Auth token for plugin installations and removal in managed instances
 install_token =
+# Comma separated list of plugin ids for which angular deprecation UI should be disabled
+hide_angular_deprecation =
```

Comment:
Going through this again, I think maybe it's worth renaming this config option to `hide_angular_deprecation_plugins` instead (with `plugins` at the end). This would be similar and more consistent to `allow_loading_unsigned_plugins` and `disable_plugins`, which are both plugin ids lists.

@andresmgot wdyt?

Reply:
Sounds good, I will keep it as-is then